src/man/pam_sss.8.xml

```diff
@@ -262,6 +262,95 @@ auth sufficient pam_sss.so allow_missing_name
         from other sources during access control.</para>
     </refsect1>
 
+    <refsect1 id="return_values">
+        <title>RETURN VALUES</title>
+        <variablelist>
+            <varlistentry>
+                <term>PAM_SUCCESS</term>
+                <listitem>
+                    <para>
+                        The user is allowed to continue authentication.
+                    </para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term>PAM_USER_UNKNOWN</term>
+                <listitem>
+                    <para>
+                        The user is not known to the authentication service or
+                        the SSSD's PAM responder is not running.
+                    </para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term>PAM_IGNORE</term>
+                <listitem>
+                    <para>
+                        See options <option>ignore_unknown_user</option> and
+                        <option>ignore_authinfo_unavail</option>.
+                    </para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term>PAM_AUTHTOK_ERR</term>
+                <listitem>
+                    <para>
+                        Unable to obtain the new authentication token. Also, could be
+                        returned when the user authenticates with certificates and
+                        multiple certificates are available, but the installed version
+                        of GDM does not support selection from multiple certificates.
+                    </para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term>PAM_AUTHINFO_UNAVAIL</term>
+                <listitem>
+                    <para>
+                        Unable to access the authentication information.
+                        This might be due to a network or hardware failure.
+                    </para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term>PAM_BUF_ERR</term>
+                <listitem>
+                    <para>
+                        A memory error occurred. Also, could be returned when options
+                        use_first_pass or use_authtok were set, but no password was
+                        found from the previously stacked PAM module.
+                    </para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term>PAM_SYSTEM_ERR</term>
+                <listitem>
+                    <para>
+                        A system error occurred.
```

Comment:
I think it would be good to say that this means an error on the SSSD side and the SSSD logs should be checked why it occured.